cmd/stack_create_test.go

```diff
@@ -214,3 +214,36 @@ func TestStackAlreadyExists(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestStackCreateMissingArguments(t *testing.T) {
```

Comment:
I think that maybe it should be made clear that the test is expecting a failure in its title `TestStackCreateFailMissingArguments` or `TestStackCreateMissingArgumentsFail`,  because the `if err == nil` sometimes throws people off when scanning through (if this test were to fail, someone might assume it's because of err is being returned as nil - generally what we'd want after running a command)